indico/modules/events/registration/templates/management/_reglist.html

```diff
@@ -38,11 +40,12 @@
                                            value="{{ registration.id }}"
                                            data-has-files="{{ registration.has_files | tojson }}">
                                 </td>
+                                {{ template_hook('registration-status-flag', regform=regform, registration=registration) }}
```

Comment:
`header=false` - like this we always call the hook with the same list of arguments